src/cascadia/TerminalSettingsModel/MTSMSettings.h

```diff
@@ -24,6 +24,7 @@ Author(s):
     X(hstring, WordDelimiters, "wordDelimiters", DEFAULT_WORD_DELIMITERS)                                                                                                                             \
     X(bool, CopyOnSelect, "copyOnSelect", false)                                                                                                                                                      \
     X(bool, FocusFollowMouse, "focusFollowMouse", false)                                                                                                                                              \
+    X(bool, DisableMouseZoom, "disableMouseZoom", false)                                                                                                                                              \

```

Comment:
```suggestion
    X(bool, ScrollToZoom, "experimental.scrollToZoom", true)                                                                                                                                              \
```
How about this name? That way we avoid the double negative. I think the `experimental` prefix is important since this setting may eventually be deprecated in favor of #1553.